test/di-order.js

```diff
@@ -15,19 +15,17 @@ var commonFalsePositives = require('./utils/commonFalsePositives');
 var eslintTester = new RuleTester();
 eslintTester.run('di-order', rule, {
     valid: [
-        'app.controller("", function($http, $q) {});',
-        'app.directive("", function($http, $q) {});',
```

Comment:
ESLint tries to resolve identifiers. I would like to use this to make sure that the following is checked:
```
var app = angular.module('');
app.controller('', function() {});
```

But the following is ignored:
```
var app = someUnknownCall();
app.controller('', function() {})
```